src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/Set-AzConnectedKubernetes.ps1

```diff
@@ -474,98 +514,79 @@ function Set-AzConnectedKubernetes {
             }
         }
 
-        if (![string]::IsNullOrEmpty($HttpProxy) -or ![string]::IsNullOrEmpty($HttpsProxy) -or ![string]::IsNullOrEmpty($NoProxy) -or ![string]::IsNullOrEmpty($HttpProxy) ) {
-            if (-not $ConfigurationSetting.ContainsKey("proxy")) {
-                $ConfigurationSetting["proxy"] = @{}
-            }
-            if (-not $ConfigurationProtectedSetting.ContainsKey("proxy")) {
-                $ConfigurationProtectedSetting["proxy"] = @{}
-            }
+        if (-not $ConfigurationProtectedSetting.ContainsKey("proxy")) {
+            $ConfigurationProtectedSetting["proxy"] = @{}
         }
 
         if (-not ([string]::IsNullOrEmpty($HttpProxy))) {
             $HttpProxyStr = $HttpProxy.ToString()
             $HttpProxyStr = $HttpProxyStr -replace ',', '\,'
             $HttpProxyStr = $HttpProxyStr -replace '/', '\/'
-            $options += " --set global.httpProxy=$HttpProxyStr"
-            $proxyEnableState = $true
-            $ConfigurationSetting["proxy"]["http_proxy"] = $HttpProxyStr
+            # $options += " --set global.httpProxy=$HttpProxyStr"
```

Comment:
Remove this comment as in theory we don't know what the mapping is because it is determined by ConfigDP.